compiler/circle-mpqsolver/src/bisection/VISQErrorApproximator.test.cpp

```diff
@@ -31,6 +31,15 @@ void writeDataToFile(const std::string &path, const std::string &data)
   file.close();
 }
 
+void makeTemporaryFile(char *name_template)
```

Comment:
maybe `const char *name_template` or `const std::string &name_template` ?

Reply:
@seanshpark unfortunately `mkstemp` requires `char *`, it changes it.
It replaces `XXXXXX` with something meaningful.